### PR TITLE
Add ability to sort failed outputs by primary key for readability

### PR DESCRIPTION
**Pull request by SOVALINUX**
In case of multiple lines of difference it's handy to order them by some "primary key" and compare 2 adjacent lines
Adding this optional parameter to the test
If you like the idea will add a note to README as well

### CODE REVIEW

File: macros/tests.sql [hunk 1]
```diff
@@ -150,9 +150,15 @@
     {{ dbt_utils.except() }}
     select '-' as diff, {{columns}} from actual)
 
-    select * from extra_entries
-    UNION ALL
-    select * from missing_entries
+    {% if options.get("primary_key", '') | length > 0 %}
```
Comment (cdiniz) [inline macros/tests.sql:153]: UNION ALL can have an order by that works on the union, then we can simplify the code: 

```
    {% set sort_field = options.get("primary_key") %}
    select * from extra_entries
    UNION ALL
    select * from missing_entries
    {% if sort_field %}
      order by {{ sort_field }}
    {% endif %}
```

It works in both Snowflake, BigQuery and Postgres.

Comment (SOVALINUX) [reply macros/tests.sql:153]: Hi
Agree that `primary_key` is not the best option
What about `output_sort_field`?

As for UNION ALL ORDER BY - Yes, great suggestion - didn't know it :) It even works in Athena I use as a primary connector

Comment (cdiniz) [reply macros/tests.sql]: I like the name, thank you!

**Review by cdiniz — APPROVED** [review]
All good and clear. Thank you.

**Pull request merged**